airbyte-integrations/connectors/source-harvest/source_harvest/manifest.yaml

```diff
@@ -0,0 +1,2832 @@
+                          api_token: "{{ config['api_token'] }}"
+                        oauth:
+                          type: OAuthAuthenticator
+                          client_id: "{{ config['client_id'] }}"
+                          grant_type: refresh_token
+                          client_secret: "{{ config['client_secret'] }}"
+                          refresh_token: "{{ config['client_refresh_token'] }}"
+                          refresh_request_body: {}
+                          token_refresh_endpoint: https://id.getharvest.com/api/v2/oauth2/token
+                    error_handler:
+                      type: CompositeErrorHandler
+                      error_handlers:
+                        - type: DefaultErrorHandler
+                          backoff_strategies:
+                            - type: WaitTimeFromHeader
+                              header: Retry-After
+                    request_body_json: {}
+                  record_selector:
+                    type: RecordSelector
+                    extractor:
+                      type: DpathExtractor
+                      field_path:
+                        - users
+                  paginator:
+                    type: DefaultPaginator
+                    page_token_option:
+                      type: RequestPath
+                    pagination_strategy:
+                      type: CursorPagination
+                      page_size: 1
+                      cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+                      stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+                  partition_router: []
+                incremental_sync:
+                  type: DatetimeBasedCursor
+                  cursor_field: updated_at
+                  cursor_datetime_formats:
+                    - "%Y-%m-%dT%H:%M:%SZ"
+                  datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+                  start_datetime:
+                    type: MinMaxDatetime
+                    datetime: "{{ config['start_date'] }}"
+                    datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+                  start_time_option:
+                    type: RequestOption
+                    field_name: updated_since
+                    inject_into: request_parameter
+    transformations:
+      - type: AddFields
+        fields:
+          - path:
+              - parent_id
+            value: "{{stream_partition.id}}"
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: updated_at
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%SZ"
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_datetime:
+        type: MinMaxDatetime
+        datetime: "{{ config['start_date'] }}"
+        datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      start_time_option:
+        type: RequestOption
+        field_name: updated_since
+        inject_into: request_parameter
+  - type: DeclarativeStream
```

Comment:
Differences observed:
* The shape of the state is different as this is a "per partition" state. This means that this will induce a breaking change and this needs to be documented

Reply:
this stream doesn't support incremental, right? What's the breaking change?

Reply:
Agreed @girarda